kolibri/plugins/coach/assets/src/views/plan/LessonSummaryPage/ManageLessonModals.vue

```diff
@@ -47,6 +47,7 @@
       // Should be 'COPY' or 'DELETE'
       currentAction: {
         type: String,
+        default: null,
```

Comment:
Can you improve this? Make it a required prop and then add a validator function. That will make the comment above unnecessary.

Reply:
okay!